runtime/doc/options.txt

```diff
@@ -3632,6 +3632,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 			|alternate-file| or using |mark-motions| try to
 			restore the |mark-view| in which the action occurred.
 
+	  unload        Remove unloaded buffers from the jumplist.
+			EXPERIMENTAL: this flag may change in the future.
```

Comment:
It's hard to think of a good name for this, but perhaps `clean` ?

Reply:
Ah yes, sounds good.